svensk-exegetisk-arsbok.csl

```diff
@@ -0,0 +1,782 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" page-range-format="chicago" demote-non-dropping-particle="sort-only" default-locale="sv-SE">
```

Comment:
I'm unsure about the function of the default locale, but I am currently unable to get Zotero 4 to produce Swedish footnotes in Word 2011 without this default locale in place. Swedish is the most common language used in this periodical, so it would make sense for a default setting (as opposed to a mandatory one).